shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -34,11 +36,21 @@ impl ResultValue {
     }
 }
 
-fn assert_query_result(session: &Session, query: Statement, expected_rows: &[&[ResultValue]]) {
-    let result = session.execute(&query).wait().unwrap();
-    let result_rows: Vec<Vec<ResultValue>> = result
+fn assert_query_result(session: &Session, query: &str, expected_rows: &[&[ResultValue]]) {
+    let result_rows = run_query(session, query);
+    assert_eq!(result_rows, expected_rows);
+}
+
+fn assert_query_result_contains(session: &Session, query: &str, row: &[ResultValue]) {
```

Comment:
hmmm I initially thought this did an unordered comparison instead of checking for the existence of a single value.
Maybe we should name it assert_query_result_contains_row to make it obvious we are checking if a single row exists?

Or maybe we should replace it with a function that actually does do an unordered comparison.
That would force developers to make stronger assertions on the returned values.

Reply:
I think checking for the existence of a row is useful, lets us keep the test queries simple. `assert_query_result` should be checking unordered though so I will add that.

Reply:
Hmmm, I think the broader we make our assertions the more cases we can catch where shotover is messing up.
But I realize that does get in the way a bit of the "test one thing" principle so I'll let you decide what you want to do here.

Reply:
Oh right, looking at the `SELECT keyspace_name FROM system_schema.keyspaces;` style queries I can see the only sane option is to check for the existence of a specific result among many.

However for the delete case we do need to check that a specific result does NOT occur.